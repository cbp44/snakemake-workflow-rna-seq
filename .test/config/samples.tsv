sample	condition	encode_experiment
bclaf1_kd_rep1	Treatment_1	ENCSR410ZPU
bclaf1_kd_rep2	Treatment_1	ENCSR410ZPU
ctrl_rep1	Control	ENCSR245BNJ
ctrl_rep2	Control	ENCSR245BNJ
