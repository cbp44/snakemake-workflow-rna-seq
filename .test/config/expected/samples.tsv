sample	condition
bclaf1_kd_rep1	Treatment_1
bclaf1_kd_rep2	Treatment_1
ctrl_rep1	Control
ctrl_rep2	Control
